### PR TITLE
Babel "include" directive is prepended to test descriptions

### PR DESCRIPTION
**Pull request by maxbarnas**
Fixes: https://github.com/ckeditor/ckeditor5-dev-compiler/issues/39.

### CODE REVIEW

File: lib/compiler/utils.js [hunk 1]
```diff
@@ -517,11 +540,18 @@ require( [ 'tests' ], bender.defer(), function( err ) {
 				);
 
 				// Convert test files.
+				const transpileTests = utils.transpile( format, utils.getBabelOptionsForTests( format, transpileToES5 ) );
+				const filterTestDescriptions = gulpFilter( ( file ) => {
+					return stream.isTestFile( file ) && file.path.endsWith( '.md' );
```
Comment (maxbarnas) [inline lib/compiler/utils.js]: The `file.path.endsWith( '.md' )` part might go at some point to `stream` repository.

File: lib/compiler/utils.js [hunk 2]
```diff
@@ -517,11 +540,18 @@ require( [ 'tests' ], bender.defer(), function( err ) {
...
+				}, { restore: true } );
 				const filterTests = gulpFilter( ( file ) => {
 					return stream.isTestFile( file ) && stream.isJSFile( file );
 				}, { restore: true } );
-				const transpileTests = utils.transpile( format, utils.getBabelOptionsForTests( format, transpileToES5 ) );
+
 				conversionPipes.push(
+					filterTestDescriptions,
```
Comment (Reinmar) [inline lib/compiler/utils.js]: It doesn't make sense to add these 3 stream processors if `transpileToES5` is `false`.

Comment (maxbarnas) [inline lib/compiler/utils.js]: That's true. But moving it to separate conditional block breaks the flow of the code. Leaving it as is might bring small performance penalty, but is less surprising. I've made the change and it certainly isn't improving readability much.

Comment (Reinmar) [inline lib/compiler/utils.js]: You can do this like this:

```js
if ( addES5 ) {
   convesionPipes.push( x, y, z );
}

conversionPipes.push( the usual );
```

**Comment by Reinmar** [comment]
The code looks good now. Missing only tests.

**Comment by maxbarnas** [comment]
Currently, test description files are processed only when format is `amd` or `cjs`. I am not sure how Bender handles it, but what about situation, when `esnext` is required?

**Comment by Reinmar** [comment]
Using `formats=esnext` + `--es5` makes very little sense, so I think this is ok.

**Pull request merged**